android-bindings/src/bindings.rs

```diff
@@ -1692,12 +1693,19 @@ pub unsafe extern "C" fn Java_com_mobilecoin_lib_TransactionBuilder_add_1output(
 
             let value = jni_big_int_to_u64(env, value)?;
 
+            // TODO: If you want to do mixed transactions, use something other
+            // than fee_token_id here.
```

Comment:
Is this TODO a note for the consumers of the android-bindings? Maybe providing a little more clarity on where to look to be able to do mixed transactions, e.g.

```suggestion
            // NOTE: This transaction builder does not support mixed transactions at this time. To 
            // implement mixed transactions, see the implementation in TransactionBuilder
            // TODO: Provide mixed-transaction functionality in these bindings.
```

Reply:
The note is for Android bindings maintainers to have guidance --

The way things work now, libmobilecoin and android have their own headers and cross language linkage matching to this code, so I can't change the API without breaking the build. When I add functionality to Tx builder, I configure it to just work the way it was working before so as not to break their stuff, but then leave a note about where that configuration is happening.

So to me this comment is confusing : "This transaction builder does not support mixed transactions at this time " because the transaction builder does support that as of this revision. It's just that the bindings still expose only the previous functionality. So the note is to the binding maintainer about how they could expose that if desired